examples/bundled/public/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0">
+    <link rel="shortcut icon" href="<%= webpackConfig.output.publicPath %>favicon.ico">
```

Comment:
What does this do? If this is common knowledge for JS devs then this is fine as is

Reply:
It is part of the output generated by vue-cli. Looks like it links to the static output dir to look for the favicon rather than the default path. I think it's fine to leave as-is – this is meant to be a project with more complexity and this is indicative of the kinds of things that will be going on in a bigger project.

Reply:
Sounds good to me.